source/addonHandler/__init__.py

```diff
@@ -508,7 +538,6 @@ def getCodeAddon(obj=None, frameDist=1):
 	@return: L{Addon} instance or None if no code does not belong to a add-on package.
 	@rtype: C{Addon}
 	"""
-	global _availableAddons

```

Comment:
This is not really related to the main problem this PR solves, but there is no reason for `_availableAddons` being global in that scope, so I've removed this line. Submitting a separate PR for a small change like this is excessive IMO.